### DISCUSSION TITLE
WineCX21.2: Available

### DISCUSSION THREAD

**Discussion by Gcenx** [The-Wineskin-Project/WineskinServer#238] [Announcements]
crossover-sources-21.2.0 has been downloaded and the wine component has been uploaded to my mirror.

Most patches were successfully rebased onto the newer sources resulting in 4 dropped patches (these are now included in CX21.2), imported some additional patches.\
Still need to review if any additional patches are worth rebasing from upstream other than the usual wine-mono bump and related commits.\
clipcurser was already included in CX21.2 by CodeWeavers.

<br>

VKD3D-macOS will be bumped to v1.3 this is only really used for WineD3D Vulkan backend for 64Bit DX10/11 when not use DXVK.

<br>

An initial WineCX21.2.0 compile was left running when I’d headed to work so hopefully the compile completes without issue and passes initial testing.

**Comment by Gcenx** [comment]
Welp it’s build and sitting in my Intel MacBook Pros desktop waiting to be packaged up unless I find anymore useful patches to apply.

It will require the usual wrapper bump but waiting on some other things to be finalized, including the idea to run my own forks of winesticks and DXVK specifically aimed at macOS. This came up due to how many users still installing the DXVK Linux verbs (those are marked Linux only btw) over running the provided custom verb.

**Reply by sofakng** [reply]
I'm looking forward to installing and testing this!

What do you mean about forks of winetricks and DXVK for macOS?  What is currently the correct way to install them?

**Reply by Gcenx** [reply]
> What do you mean about forks of winetricks and DXVK for macOS?

On GitHub you can “fork” a repository aka make a copy usually that’s done if you want to contribute back to the original project or make a modified version.

> What is currently the correct way to install them?

Wineskin will now obtain winetricks from my fork the first time the winetricks menu is opened, or you click the “update” button.

DXVK-macOS can be installed directly from the winetricks menu the supported version mention Mac in the description.

The only supported versions of DXVK-macOS 1.9 > 1.10.1 for now, I don’t see myself building older versions.

**Comment by Gcenx** [comment]
The CX21.2.0 Engines we’re pushed
